### PR TITLE
Update pt-BR.txt

### PR DESCRIPTION
**Pull request by renansimoes**
If you are applying translations for an issue (or multiple), please list them below (if not, delete this text).

Translated new strings, modified some others (acording to the issue), and moved RCT2 rides descriptions strings to the end.

Applying for issue(s):
- #1400
- #1411
- #1419
- #1424
- #1432
- #1442
- #1446
- #1450
- #1451
- #1454
- #1455
- #1461
- #1465

### CODE REVIEW

File: data/language/pt-BR.txt [hunk 1]
```diff
@@ -2907,7 +2907,7 @@ STR_2896    :{WINDOW_COLOUR_2}Hypothermia:   (Allister Brimble) direitos autorai
 STR_2897    :{WINDOW_COLOUR_2}Last Sleigh Ride:   (Allister Brimble) direitos autorais {COPYRIGHT} Chris Sawyer
 STR_2898    :{WINDOW_COLOUR_2}Pipes of Glencairn:   (Allister Brimble) direitos autorais {COPYRIGHT} Chris Sawyer
 STR_2899    :{WINDOW_COLOUR_2}Traffic Jam:   (Allister Brimble) direitos autorais {COPYRIGHT} Chris Sawyer
-STR_2900    :{WINDOW_COLOUR_2}
+STR_2900    :{WINDOW_COLOUR_2}<removed string - do not use>
```
Comment (Gymnasiast) [inline data/language/pt-BR.txt]: This line should not have been changed. Could you revert this line?

Comment (renansimoes) [reply data/language/pt-BR.txt]: Ok, I changed because in En-gb file has this "<removed string - do not use>", but ok, I'll revert it.

Comment (AaronVanGeffen) [reply data/language/pt-BR.txt]: That is the case, yes, but en-GB has the formatting code *replaced* with the placeholder string. The  `{WINDOW_COLOUR_2}` part should be removed.

Comment (renansimoes) [reply data/language/pt-BR.txt]: ok

Comment (AaronVanGeffen) [reply data/language/pt-BR.txt]: Oh, I didn't realise I was replying to an outdated comment. Sorry for the confusion. I believe you were right to do the initial change, but the line should read the same as it does in English, being:
```
STR_2900    :<removed string - do not use>
```

Comment (renansimoes) [reply data/language/pt-BR.txt]: Errr, so I should add " :<removed string - do not use>" or not?

**Pull request merged**